dev/io.openliberty.jakarta.jstl.2.0/bnd.bnd

```diff
@@ -21,13 +21,13 @@ Export-Package: \
 	jakarta.servlet.jsp.jstl.tlv
 
 Import-Package:  jakarta.servlet.jsp.*;version="[2.0.0,4.0.0)", \
- jakarta.servlet.*; version="[5.0.0,6.0.0)", \
+ jakarta.servlet.*; version="[5.0.0,7.0.0)", \
```

Comment:
@jhanders34 can you explain to me why we're expanding the EE9 bundle imports to include the EE10 version range? I see similar changes for pages 3.0 and websocket 2.0.

Reply:
Because there isn't a 3.1 version of the jakarta.jstl yet, so to allow the feature to start, I temporarily did this in order to allow for the features to still start until the jstl function is added.  I know that jstl, faces, and maybe some of the other ones we do NOT use the jakarta. versions of the API bundles so until those are brought in I did this temporarily to be able to move forward until the jstl API is brought in.  If you say that we are going to use the jstl from jakarta this time I can make the io.openliberty.jakarta/tags.bnd file and just reference it there.  Right now for 3.0 it appears we use the 2.0 one and transform it which is super odd.  If you want me to do things differently, I can.

Reply:
We did the transformation for the Expression Language to resolve as a temp solution. We have a number of PRs in flight to add the new API bundles for Jakarta Tags and WebSockets . How are we tracking these as temporary solutions? Do you have another issue open to undo these changes?  I just want to ensure that we end up with the correct solution in the end.

PRs: https://github.com/OpenLiberty/open-liberty/pull/21442 / https://github.com/OpenLiberty/open-liberty/pull/21099

Which feature isn't starting and required this change?

Reply:
Also as far as I can tell we did use the Jakarta Tags 2.0 API jar from the Eclipse community for EE9: 

 Eclipse Jakarta Tags Project: https://github.com/eclipse-ee4j/jstl-api/blob/2.x/api/pom.xml#L31 

 OL pages-3.0 feature: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/io.openliberty.pages-3.0.feature#L41

 Jakarta Tags 2.0 OL API bndle: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/io.openliberty.jakarta.jstl.2.0/bnd.bnd#L37

Reply:
The tags transform is what threw me for a loop.  I have now added tags and pages to the list of API bundle being created.